From 5fe93cff56f73791e22e6dde57f88419aa0f174d Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh,wz@renesas.com>
Date: Tue, 28 May 2019 10:46:27 +0700
Subject: [PATCH 160/248] arm64: dts: renesas: r8a774a1: Add USB3.0 host

device node

Add USB3.0 (XHCI) host device node to SoC R8A774A1

Signed-off-by: Huynh Thanh Hung <hung.huynh,wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
index 0e7b4a1..5af2f9a 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
@@ -1464,6 +1464,17 @@
 			resets = <&cpg 318>;
 			status = "disabled";
 		};
+
+		xhci0: usb@ee000000 {
+			compatible = "renesas,xhci-r8a774a1",
+				     "renesas,rcar-gen3-xhci";
+			reg = <0 0xee000000 0 0xc00>;
+			interrupts = <GIC_SPI 102 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD 328>;
+			power-domains = <&sysc R8A774A1_PD_ALWAYS_ON>;
+			resets = <&cpg 328>;
+			status = "disabled";
+		};
 	};
 
 	thermal-zones {
-- 
2.7.4

